README.md

```diff
@@ -23,6 +23,15 @@ Anyone is welcome to contribute to Apollo Server, just read [CONTRIBUTING.md](./
 
 Apollo Server is super-easy to set up. Just npm-install apollo-server, write a GraphQL schema, and then use one of the following snippets to get started. For more info, read the [Apollo Server docs](http://docs.apollostack.com/apollo-server).
 
+### TypeScript project
+
+To build your GraphQL server using TypeScript, Apollo Server is the project for you.  To add Apollo to your project:
+
+```sh
+npm install apollo-server
+typings i -SG dt~express dt~express-serve-static-core dt~serve-static dt~mime dt~hapi dt~cookies dt~koa
+```
+
```

Comment:
I think it's worth mentioning even if it's just one line. If there's a section "setup with X" and I'm using Y, then I will look for a section "setup with Y". Since we're not short on space, I think it's fine to be verbose but clear. Or do you think it somehow muddles the TypeScript message?